rest-api/participant_enums.py

```diff
@@ -37,6 +37,11 @@ class SampleStatus(messages.Enum):
   UNSET = 0
   RECEIVED = 1
 
+class OrderStatus(messages.Enum):
+  """Status of biobank orders and samples"""
+  UNSET = 0
+  ORDERED = 1
```

Comment:
Consider expanding this to include `UNSET`, `CREATED`, `COLLECTED`, `PROCESSED`, `FINALIZED` -- note that the "realtime" nature of these reports may be a bit misleading (let's discussion on Slack).

Reply:
Done.